pkg/controller/stackconfigpolicy/controller.go

```diff
@@ -233,6 +235,13 @@ func (r *ReconcileStackConfigPolicy) doReconcile(ctx context.Context, policy pol
 		return results.WithError(err), status
 	}
 
+	// check for weight conflicts with other policies
+	if err := r.checkWeightConflicts(ctx, &policy); err != nil {
```

Comment:
Should this be part of the admission webhook validation checks already to give immediate feedback to users?